nixos/modules/system/boot/loader/generic-extlinux-compatible/extlinux-conf-builder.sh

```diff
@@ -75,9 +75,8 @@ addEntry() {
 
     copyToKernelsDir "$path/kernel"; kernel=$result
     copyToKernelsDir "$path/initrd"; initrd=$result
-    # XXX UGLY: maybe the system config should have a top-level "dtbs" entry?
-    dtbDir=$(readlink -m "$path/kernel/../dtbs")
-    if [ -d "$dtbDir" ]; then
+    dtbDir=$(readlink -m "$path/dtbs")
+    if [ -e "$dtbDir" ]; then
```

Comment:
(Probably nothing to change) this is because it might be a symlink I presume?

Reply:
Initially I had planned that this might be a single dtb file - but then uboot wouldn't be able to find it so that might be pointless.

Reply:
I've undone this change.

Reply:
I think this change is necessary for the bootloader module to pick up the modified files. On my system kernel is a symlink.

Reply:
Good point, re-applied.